app/src/main/java/com/android/unio/ui/authentication/EmailVerification.kt

```diff
@@ -0,0 +1,96 @@
+package com.android.unio.ui.authentication
+
+import android.widget.Toast
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Refresh
+import androidx.compose.material3.Button
+import androidx.compose.material3.Icon
+import androidx.compose.material3.OutlinedButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.android.unio.ui.navigation.NavigationAction
+import com.android.unio.ui.navigation.Screen
+import com.android.unio.ui.theme.AppTypography
+import com.google.firebase.Firebase
+import com.google.firebase.auth.auth
+
+@Composable
+fun EmailVerificationScreen(navigationAction: NavigationAction) {
+
+  val user by remember { mutableStateOf(Firebase.auth.currentUser) }
+  var success by remember { mutableStateOf(false) }
+
+  val context = LocalContext.current
+
+  val checkEmailVerification = {
```

Comment:
We can add a `Log` and/or a `Toast` here., in case the refresh is not successful.